srv/salt/ceph/mgr/keyring/default.sls

```diff
@@ -1,13 +1,10 @@
-
-
 /var/lib/ceph/mgr/ceph-{{ grains['host'] }}/keyring:
   file.managed:
     - source:
       - salt://ceph/mgr/cache/{{ grains['host'] }}.keyring
     - template: jinja
-    - user: ceph
-    - group: ceph
+    - user: root
+    - group: root
```

Comment:
This state copies the cached/temp keyring over to another minion. Before we used containers, ceph was installed on every host, making the `ceph` user available on the system. As we're now in the container land, the ceph user is only present *inside* the container. Since it's salt that tries to change the ownership of this file, it fails with UserNotFound (or something like that :)). My only idea to keep the ceph:ceph permissions, is to change the ownership from inside the container. Not sure what the security(?) implications are when running as root.

This state is probably the only one where we have to operate outside of a container.